index.js

```diff
@@ -60,7 +61,9 @@ function exorcist(file, url, root, base) {
       );
       return write(src.source);
     }
-
+    // make sure the file folder exists
+    mkdirp.sync(path.dirname(file));
```

Comment:
Can't have sync calls in a lib (people may use it as such even if main use case is bin tool).
Please find a way to do this async.